### PR TITLE
Refactor the documentation for compatibility with the docs.scs.community

### PR DESCRIPTION
**Pull request by matofeder**

### CODE REVIEW

File: docs/kaas.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+To test the Monitoring of the KaaS layer use case, deploy the Kaas-metric-importer
+into the Observer cluster.
+
+The Kaas-metric-importer is a simple service through which the KaaS software registers
+and unregisters newly created or deleted KaaS clusters in the Observer monitoring.
+This functionality enables the Observer monitoring to differentiate between KaaS clusters deleted intentionally
+and those that have stopped writing metrics to the Observer monitoring for any reason.
+
+```bash
+kubectl apply -f ../kaas-metric-importer.yaml
```
Comment (chess-knight) [inline docs/kaas.md:15]: We can move `kaas-metric-importer.yaml` to the `kaas/`
```suggestion
kubectl apply -f kaas/kaas-metric-importer.yaml
```

File: README.md [hunk 2]
```diff
@@ -1,182 +1,13 @@
...
-with label `cluster` and value `1` into thanos receiver.
-E.g. configmap:
-```yaml
-data:
-  workload-cluster: ""
-```
-It pushes metric e.g. `kaas{cluster="workload-cluster"} 1` to the Observer.
-
-
-### Optional: Monitoring of infrastructure services deployment
```
Comment (chess-knight) [inline README.md:108]: Where is the documentation about this option?

File: docs/kaas.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+http://127.0.0.1:8080/kaas
+```
+
+- Create KaaS cluster through Swagger UI: [Create Cluster](http://127.0.0.1:8080/kaas#/Clusters/create_cluster_api_clusters__post) or
+  call directly the KaaS service API via some client, e.g.:
+  ```bash
+  curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8080/api/clusters/ -d '{"name": "kaas"}'
+  ```
+
+  Navigate to the [KaaS Monitoring dashboard](http://localhost:30000/d/kaas-monitoring/kaas-monitoring)
```
Comment (chess-knight) [inline docs/kaas.md:70]: Shouldn't we instruct users to enable kaasMonitoring? by. e.g. values of observer cluster:
```bash
dnation-kubernetes-monitoring:
  kaasMonitoring:
    enabled: true
```
Also, other stuff is needed there, like `receive.enabled=true` and optional KubernetesMonitoringClusterDown alert and `--query.lookback-delta=1m` flag.

Maybe point the user to MVP0 comments...

File: README.md [hunk 4]
```diff
@@ -1,182 +1,13 @@
...
-Refer to [kaas README file](./kaas/README.md).
-
-## Monitoring of infrastructure services (container registry)
-
-Refer to [registry README file](./registry/README.md).
-
-## Forwarding of alerts into Matrix chat
-
-Refer to [matrix-alertmanager README file](./matrix-alertmanager/README.md).
+Explore the documentation stored in the [docs](./docs) directory or view the rendered version online at https://docs.scs.community/docs/container/.
```
Comment (chess-knight) [inline README.md:13]: This is part of the Ops layer, not the Container

File: README.md (hunk 2 above)
Comment (matofeder) [reply README.md:108]: thank you for pointing this out; here it is: 51f377fd2350183356dd8e625b8c0a83ebfd9b7e

File: docs/kaas.md (hunk 3 above)
Comment (matofeder) [reply docs/kaas.md:79]: good point: 
temporarily I point the reader to the mvp-0 tag, I believe that the TODO comment will be addressed within the stabilization work on the KaaS layer 

see 6ca5607469c7fdbd15c8d9398b1188ff518ad595

**Review by chess-knight — APPROVED** [review]

**Pull request merged**